src/confcom/azext_confcom/README.md

```diff
@@ -37,66 +39,88 @@ Install the Azure CLI and Confidential Computing extension.
 
 See the most recently released version of `confcom` extension.
 
-    az extension list-available -o table | grep confcom
+```bash
+az extension list-available -o table | grep confcom
+```
 
 To add the most recent confcom extension, run:
 
-    az extension add --name confcom
+```bash
+az extension add --name confcom
```

Comment:
You could also add the update command here for when the users already have a version installed